lib/src/widgets/chat/ds_end_calls_message_bubble.widget.dart

```diff
@@ -0,0 +1,272 @@
+  }) : style = style ?? DSMessageBubbleStyle();
+
+  @override
+  Widget build(BuildContext context) {
+    return DSMessageBubble(
+      borderRadius: borderRadius,
+      align: align,
+      style: style,
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              Column(
+                children: [
+                  Row(
+                    children: [
+                      Padding(
+                        padding: const EdgeInsets.only(right: 8.0),
+                        child: Column(
+                          children: [
+                            Container(
+                              decoration: BoxDecoration(
+                                color: _isCallAnswered
+                                    ? DSColors.success
+                                    : DSColors.error,
+                                borderRadius: const BorderRadius.all(
+                                  Radius.circular(
+                                    8.0,
+                                  ),
+                                ),
+                              ),
+                              width: 30,
+                              height: 30,
+                              child: Icon(
+                                _isCallAnswered
+                                    ? _isInbound
+                                        ? DSIcons.voip_receiving_outline
+                                        : DSIcons.voip_calling_outline
+                                    : DSIcons.voip_ended_outline,
+                                size: 20.0,
+                              ),
+                            ),
+                          ],
+                        ),
+                      ),
+                      Column(
+                        crossAxisAlignment: CrossAxisAlignment.start,
+                        children: [
+                          DSCaptionText(
+                            'calls.voice-text'.translate(),
+                            fontWeight: FontWeight.bold,
+                            color: _foregroundColor,
+                          ),
+                          DSCaptionSmallText(
+                            _isCallAnswered
+                                ? 'calls.answered'.translate()
+                                : 'calls.unanswered'.translate(),
+                            color: _foregroundColor,
+                          ),
+                        ],
+                      ),
+                    ],
+                  ),
+                ],
+              ),
+              Flexible(
+                child: Column(
+                  children: [
+                    FutureBuilder(
+                      future:
+                          content['identification'].toString().asPhoneNumber(),
```

Comment:
This async process should begin in `initState` method to avoid re-executing it whenever the widget renders as described by the article below:

https://yasinefemdalkilic.medium.com/flutter-dont-use-function-in-future-builder-d94314fdcb70